CHANGELOG.md

```diff
@@ -13,6 +13,28 @@ next release
 
 ### UI Changes
 
+1.34.0 (2022-05-11)
+-------------------
+### Backend Changes
+
+#### Breaking Changes
+
+#### New Features
+
+* Support writing via gRPC stream in storage plugin ([@vuuihc](https://github.com/vuuihc) in [#3640](https://github.com/jaegertracing/jaeger/pull/3640))
+* Instrument MetricsReader with metrics ([@albertteoh](https://github.com/albertteoh) in [#3667](https://github.com/jaegertracing/jaeger/pull/3667))
+
+#### Bug fixes, Minor Improvements
+
+* Fix es.log-level behaviour ([@albertteoh](https://github.com/albertteoh) in [#3664](https://github.com/jaegertracing/jaeger/pull/3664))
+* Mention remote gRPC storage API ([@yurishkuro](https://github.com/yurishkuro) in [cb6853d](https://github.com/jaegertracing/jaeger/commit/cb6853d4eea1ab5430f5e8db6b603cd7de5a16c3))
+* Perform log.fatal if TLS flags are used when tls.enabled=false #2893 ([@clock21am](https://github.com/clock21am) in [#3030](https://github.com/jaegertracing/jaeger/pull/3030))
+* Upgrade to Go 1.18 ([@yurishkuro](https://github.com/yurishkuro) in [#3624](https://github.com/jaegertracing/jaeger/pull/3624))
```

Comment:
it looks like we had a slew of other changes merged since this started, do you want to include them? Once this is merged it will be at the tip and implicitly include those changes.

Reply:
I have added the missing changes.